lib/module.js

```diff
@@ -1,32 +1,24 @@
 const { resolve } = require('path')
 
-function nuxtFathom(options) {
-  if ( typeof options.auto !== 'boolean' ) {
-    options.auto = true
-  }
-
-  if ( typeof options.canonical !== 'boolean' ) {
-    options.canonical = true
-  }
-
-  if ( !options.excludedDomains ) {
-    options.excludedDomains = ''
-  }
-
-  if ( typeof options.honorDoNotTrack !== 'boolean' ) {
-    options.honorDoNotTrack = false
-  }
-
-  if ( !options.includedDomains ) {
-    options.includedDomains = ''
-  }
-
-  if ( !options.spa ) {
-    options.spa = 'auto'
-  }
-
-  if ( !options.scriptSrc ) {
-    options.scriptSrc = 'https://cdn.usefathom.com/tracker.js'
+function nuxtFathom(moduleOptions) {
+  const { nuxt } = this
+
+  const defaultOptions = {
+    auto: true,
+    canonical: true,
+    excludedDomains: '',
+    honorDoNotTrack: false,
+    includedDomains: '',
+    spa: 'auto',
+    scriptSrc: 'https://cdn.usefathom.com/tracker.js'
+  }
+
+  // Combine options
+  const options = {
+    ...defaultOptions,
+    ...nuxt.options.fathom,
+    ...moduleOptions,
+    ...(nuxt.options.publicRuntimeConfig && nuxt.options.publicRuntimeConfig.fathom)
```

Comment:
Module itself is runtime. We should use runtime-config directly inside plugin. Example: https://github.com/nuxt/http/blob/658b8822e5c24ecf5eb6207a25cf150b1f88bcc5/lib/plugin.js#L190